.travis.yml

```diff
@@ -1,32 +1,26 @@
 language: python
 matrix:
-  allow_failures:
-    - env: TOX_ENV=squeeze
   include:
-    - python: 2.6
-      env: TOX_ENV=py26
     - python: 2.7
       env: TOX_ENV=py27
-    - python: 3.3
-      env: TOX_ENV=py33
     - python: 3.4
       env: TOX_ENV=py34
     - python: 3.5
       env: TOX_ENV=py35
     - python: 3.6
       env: TOX_ENV=py36
-    - python: 2.6
-      env: TOX_ENV=squeeze
     - python: 2.7
       env: TOX_ENV=wheezy
 install:
   - pip install -U pip
-  - pip install tox
-  - pip install pyflakes
+  - pip install flake8 tox
   - sudo rm -rf /dev/shm && sudo ln -s /run/shm /dev/shm
 script:
   - tox -e $TOX_ENV
-  - python -m pyflakes.__main__ $(git ls-files | grep -E "^isso/.+.py$" | grep -v "^isso/compat.py")
+  # stop the build if there are Python syntax errors or undefined names
```

Comment:
Please keep the move to flake8 as a separate change. This is different from just dropping Python2.6/3.3 support, and changes what kind of syntax errors will be fatal.